### PR TITLE
Monthlyexpend

### PR DESCRIPTION
**Pull request by AjeyAshok**
Added feature to add up the total expenditure for a specified month.

### CODE REVIEW

File: src/main/java/executor/command/CommandGetMonthlySpending.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package executor.command;
+
+import executor.task.TaskList;
+import interpreter.Parser;
+import ui.ReceiptTracker;
+import ui.Wallet;
+
+public class CommandGetMonthlySpending extends Command {
```
Comment (Mudaafi) [inline src/main/java/executor/command/CommandGetMonthlySpending.java]: Consider refactoring the class to _CommandGetSpendingByMonth_ or _CommandGetExpenditureByMonth_ for greater naming clarity.

File: src/main/java/executor/command/CommandGetMonthlySpending.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    @Override
+    public void execute(TaskList taskList) {
+
+    }
+
+    @Override
+    public void execute(Wallet wallet) {
+        ReceiptTracker monSpend = new ReceiptTracker();
```
Comment (Mudaafi) [inline src/main/java/executor/command/CommandGetMonthlySpending.java]: Consider refactoring to _receiptsInMonth_ or equivalent for greater code clarity.

File: src/main/java/executor/command/CommandGetMonthlySpending.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        String mon = Parser.parseForPrimaryInput(CommandType.EXPENDED, userInput);
+        int month = getMon(mon);
+        if (month != 0) {
+            int year = Integer.parseInt(Parser.parseForFlag("year", userInput));
+            monSpend = wallet.getReceipts().findReceiptByMonthYear(month, year);
+            Double totalMoney = monSpend.getTotalCashSpent();
+            System.out.println("The total amount of money spent in " + mon + " " + year + " : " + totalMoney);
+        } else {
+            System.out.println("Invalid input, CORRECT FORMAT : expended <month> <year> ");
```
Comment (Mudaafi) [inline src/main/java/executor/command/CommandGetMonthlySpending.java]: use _Ui.dukeSays_

File: src/main/java/executor/command/CommandGetMonthlySpending.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        case "september":
+            return 9;
+        case "october":
+            return 10;
+        case "november":
+            return 11;
+        case "december":
+            return 12;
+        default:
+            return 0;
```
[the remark this answers is not in the document]
Comment (AjeyAshok) [reply src/main/java/executor/command/CommandGetMonthlySpending.java]: The issue is that my function returns an INT and thus I am not able to return a string.

**Pull request merged**